10376.xml

```diff
@@ -0,0 +1,211 @@
+...
+n. array(Record Index n, Record Timestamp n, Record Value n)
+
+Record Timestamp is a 32-bit integer representing the number of seconds since Jan 1st, 1970 in the UTC time zone. Record Value can be any type and described by CBOR type. Value can also be an array of sub-values.]]></Description>
+			</Item>
+			<Item ID="7">
+				<Name>Record Head Index</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource represents the index of the latest record. This index is incremented upon every new activity's record. Total recordings inside the device can be calculated by Record Head Index - Record Tail Index.]]>
+				</Description>
+			</Item>
+			<Item ID="8">
+				<Name>Record Tail Index</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource represents the index of the oldest record stored in device. This index is adjusted (incremented) by device based on its memory space availability.]]>
+				</Description>
+			</Item>
+			<Item ID="9">
+				<Name>Record Read Index</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource represents the index of the last read record. Device will increment this index upon successful record delivery to server(via manual read or scheduled dispatch). Server may modify this index to traverse within available records in device.]]>
+				</Description>
+			</Item>	
+			<Item ID="10">
+				<Name>Record Dispatch Start Mask</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource represents the start time to deliver Record resource. If Record resource is currently being observed, this resource enables accumulation of the Record changes before being dispatch (rather than delivering Record immediately as it changes). As per Start Mask, Dispatch Start Mask must adhere to following string format:
+"YYMMDDHHmmss".
+
+Dispatch Start Mask is adjusted to UTC offset provided by Device [/3/0/14] object instance resource, or adjusted to UTC if the resource is not provided.
+
+If this resource is used together with Dispatch Interval resource, it is possible to automatically deliver Record resource at specific time. I.e.  dispatch accumulated record every 6 hours, at minute 5 (00:05, 06:05, 12:05, 18:05).
```

Comment:
```suggestion
If this resource is used together with Dispatch Interval resource, it is possible to automatically deliver Record resource at specific time. For example, dispatch accumulated record every 6 hours, at minute 5 (00:05, 06:05, 12:05, 18:05).
```

The last part of description looks like an example instead of clarification